### ISSUE TITLE
Bug: heading levels

### ISSUE THREAD

**Issue by desigonz** [cityofaustin/techstack#1287]
Some accessibility heading things:
- [ ] Missing h1
- [ ] Skipped heading levels (ex. h2 skips to h4)

**URL**
https://forms.austintexas.io/police-oversight/complaint/introduction

![image.png](https://images.zenhubusercontent.com/5a31a50b8a75884b908b0ea6/864c6f57-d20e-47e1-a4f3-839abb847218)

**Comment by easherma** [comment]
we'll probs need to discuss this and maybe just remap the heading tags
